__tests__/Game.test.js

```diff
@@ -0,0 +1,75 @@
+import Car from '../src/services/Car.js';
+import Game from '../src/services/Game.js';
+
+describe('게임(자동차 경주) 진행에 대한 테스트', () => {
+  const DEFAULT_CAR_NAMES = ['아반떼', '스타렉스'];
+  const DEFAULT_CAR_LOCATION = 0;
+  const DEFAULT_GAME_LAP = 5;
+
+  let game = null;
+
+  beforeEach(() => (game = new Game({ names: DEFAULT_CAR_NAMES })));
+
+  // Model
+  test('게임을 생성할 때 names에 ["아반떼", "스타렉스"]를 넣으면 ["아반떼", "스타렉스"] 자동차는 게임에 참가된다.', () => {
+    const players = game.getPlayers();
+
+    expect(players).toBe(DEFAULT_CAR_NAMES);
+  });
+
+  test('게임을 생성할 때, 다섯 바퀴로 기본 설정이다.', () => {
```

Comment:
테스트명을 보고 전체적인 맥락을 파악할 수 있도록 테스트명이 조금 더 친절해졌으면 좋겠습니다!

```suggestion
  test('게임을 생성할 때, 경주 횟수를 정의하지 않으면 게임의 경주 횟수는 5회로 생성되어야 한다.', () => {
```

Reply:
이 코멘트는 비단 이 부분 뿐만 아니라 전체적인 테스트코드에도 동일하게 말씀 드리고 싶은 부분인데요.

test를 실행하실 때 `--verbose` 같은 옵션을 주고 테스트 수행결과를 본 다음, 자동차 경주 게임의 도메인 지식이 없고, 맥락을 모르는 사람이 봐도 어떤 걸 테스트 하는지 알 수 있게 작성하도록 해보시면 좋을거 같습니다 !